packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -233,6 +233,15 @@ export class Fixture {
   restore() {
     window.location.hash = '#';
     localStorage.clear();
+    console.log('🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥🎥');
+    console.log(Array.from(document.getElementsByTagName('video')).length);
+
+    const videos = Array.from(document.getElementsByTagName('video'));
+    if (videos.length) {
+      videos.map((video) => video.parentNode.removeChild(video));
+    }
+    console.log('🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻🕵🏻');
+    console.log(Array.from(document.getElementsByTagName('video')).length);
```

Comment:
Sounds like you're trying to fix those video player messages?

Check out #9944 for context, especially the discussion at https://github.com/google/web-stories-wp/pull/9944#discussion_r777486850


It should not cause test failures; it's just noisy.

Reply:
Yeah at this point we are unsure where the karma test suite disconnect is occurring. There was a slim hope that the memory leak from these would cause the thread to run out of memory and thus diconnect. But this isn't the case. Seeing as the webMediaPlayer not being garbage collected is an issue with Chrome, I am not sure we can really solve this one. Max is looking into it though.